packages/storefront-lwc/src/modules/commerce/productDetail/__tests__/__snapshots__/productDetail.test.js.snap

```diff
@@ -0,0 +1,51 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<commerce-product-detail> should render carousel 1`] = `
```

Comment:
These are snapshot tests. The html of a portion of the component is saved in this snapshot, and when the tests runs in the future it verifies that it still produces the same HTML.